bigtop-packages/src/deb/hive/rules

```diff
@@ -60,6 +60,7 @@ override_dh_auto_install: server2 metastore hcatalog-server webhcat-server
 	ln -s /usr/lib/hbase/hbase-common.jar /usr/lib/hbase/hbase-client.jar /usr/lib/hbase/hbase-hadoop-compat.jar /usr/lib/hbase/hbase-hadoop2-compat.jar debian/tmp/usr/lib/hive/lib
 	ln -s /usr/lib/hbase/hbase-procedure.jar /usr/lib/hbase/hbase-protocol.jar /usr/lib/hbase/hbase-server.jar debian/tmp/usr/lib/hive/lib/
 	ln -s /usr/lib/zookeeper/zookeeper.jar debian/tmp/usr/lib/hive/lib
+        ln -s /usr/lib/hadoop//tools/lib/hadoop-distcp*.jar debian/tmp/usr/lib/hive/lib/
```

Comment:
You must use tab instead of whitespace since the rules file is written in Makefile format. @lvkaihua 
```
debian/rules:63: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.
dpkg-buildpackage: error: fakeroot debian/rules clean subprocess returned exit status 2

> Task :hive-sdeb FAILED
```

Reply:
![微信截图_20230919174412](https://github.com/apache/bigtop/assets/24928039/dea5a010-eeb2-4c51-b7bf-900a05f34ca0)
Sorry, I forgot to fix it here. I also found this issue during ubuntu testing earlier. I just made the necessary modifications and this is a screenshot of the modified pull test